app/_hub/kong-inc/opa/0.1.x.md

```diff
@@ -0,0 +1,314 @@
+---
+name: OPA
+publisher: Kong Inc.
+version: 0.1.x
+# internal handler v 0.1.0
```

Comment:
This was bumped to 0.2.0 internally.

Reply:
Probably also need to rename the file.

Reply:
the index file is the most recent version of the plugin - and this file has the 0.2.x version info in it